src/views/call-details/components/NewInjectionStreamDrawer.tsx

```diff
@@ -99,7 +99,7 @@ const NewInjectionStreamDrawer: React.FC = () => {
       mode: state.streamMode || StreamMode.Caller,
       streamUrl: state.injectionUrl,
       latency: state.latency,
-      streamKey: state.passphrase,
+      streamKey: StreamProtocol.SRT ? state.passphrase : undefined,
       enableSsl: state.enableSsl,
       keyLength: state.passphrase ? state.keyLength : KeyLength.None,
```

Comment:
**question (blocking)**: Should we also update the condition to set the `keyLength` property? What about other properties like `latency` and `enableSsl`?

Reply:
you are right, I'm sorry